main.go

```diff
@@ -95,6 +96,37 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+func SetupLogging(opts *zap.Options) {
+	if logLevelArg, ok := common.FindArgFromEnv("ARGS", "--zap-log-level="); ok {
```

Comment:
I don't think we need to validate arguments, we could just append/replace all them. If I pass a bad zap log level the operator process already fails, i.e.
```
spec:
      - args:
        - --zap-log-level=brusdev
```
```
Executing entrypoint
exec /home/activemq-artemis-operator/bin/activemq-artemis-operator
invalid value "brusdev" for flag -zap-log-level: invalid log level "brusdev"
Usage of /home/activemq-artemis-operator/bin/activemq-artemis-operator:
  -health-probe-bind-address string
...
```
It should be up to `flag.Parse()` validate args.

Reply:
+1